model/distro/distro.go

```diff
@@ -738,6 +746,10 @@ func (d *Distro) GetResolvedPlannerSettings(s *evergreen.Settings) (PlannerSetti
 	if resolved.GenerateTaskFactor == 0 {
 		resolved.GenerateTaskFactor = config.GenerateTaskFactor
 	}
+
+	// StepbackTaskFactor is always undefined in distro
```

Comment:
since we're removing it from distro entirely, we can maybe say "isn't configurable by distro"

Reply:
done

Reply:
done